src/main/resources/assets/forge/lang/en_us.json

```diff
@@ -178,5 +178,7 @@
   "generic.reachDistance": "Reach Distance",
 
   "fluid.minecraft.milk": "Milk",
-  "fluid.minecraft.flowing_milk": "Milk"
+  "fluid.minecraft.flowing_milk": "Milk",
+
+  "forge.experimentalsettings.tooltip": "This world uses experimental settings."
```

Comment:
Not a full change request, but we might want to consider giving the warning tooltip a bit more bite, to fully qualify the possibility that the world may stop working due to its use of experimental settings. Possibly something like the first sentence from `selectWorld.backupWarning.experimental`: `"This world uses experimental settings that could stop working at any time."`